src/components/SliderHero/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, Container, Flex, Heading, Stack, Text } from '@chakra-ui/react'
+
+import { ChakraNextImage } from '@components'
+
+interface SliderHeroProps {
+  items: SubpageType[] | undefined
+  index: number
```

Comment:
You don't use  neither `index` nor `items` independently in the component. Wouldn't it be better to just provide `item` in terms of repeating or unit testing.